src/System.Diagnostics.Process/tests/ProcessTest_ConsoleApp/ProcessTest_ConsoleApp.cs

```diff
@@ -7,6 +8,7 @@ class Program
     {
         static int Main(string[] args)
         {
+            Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
             try
```

Comment:
Shouldn't we fix the tests as Wes outlines and then not lock the culture to en-US? (Regardless, @Alexx999, thank you for taking the time to create a fix here.)